### PR TITLE
Refactor "Seek Feedback"

### PR DESCRIPTION
**Pull request by johnboxall**
Getting feedback is important that we should explain it more :)

I'm seeking feedback on seeking feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,14 +200,33 @@ repetitive tasks could be scripted.
...
+
+> In medicine there is a tradition, where 👩🏾‍⚕️ doctors, no matter how
+> experienced, ask colleagues for a _consult_: an opportunity to hear another
+> perspective on how to resolve a challenge.
+
+_Mobify has blatantly stole this great tradition._
+
+**Actively seek feedback and incorporate it into your work.**
+
+Before writing code, discuss solutions with your peers.
```
Comment (jansepar) [inline README.md]: id specify something like "Before writing code for large changes or entirely new features/libraries/services".

I don't think people should discuss solutions for all the code they write.

Comment (johnboxall) [inline README.md]: Now: 

> Before implementing a major change, discuss solutions with your peers.

**Comment by jansepar** [comment]
👍. New rule - all changes to Developer Values must be communicated at Eng Meeting :D

How would you feel about presenting this at the next one?

**Comment by johnboxall** [comment]
This change was presented at the Engineering Meeting here:

https://youtu.be/vTnXHkcZ-A8?t=2608

**Pull request merged**